xnmt/input.py

```diff
@@ -114,16 +141,22 @@ class ContVecReader(InputReader, Serializable):
   def __init__(self):
     pass
 
-  def read_sents(self, filename, max_num=None):
+  def read_sents(self, filename, filter_ids=None):
     npzFile = np.load(filename)
```

Comment:
Does this end up loading the whole corpus into memory? If so, maybe we need to add `mmap_mode="r"`?